src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -2367,6 +2380,16 @@ private void doSignalWriteIdleTimeoutIfNecessary()
             }
         }
 
+        private void doSignalCloseTimeout()
+        {
+            closeTimeoutAt = System.currentTimeMillis() + CLOSE_TIMEOUT;
```

Comment:
This can be a local variable, no need to hold on to it in instance state, agree?